modules/gdscript/gdscript_parser.cpp

```diff
@@ -82,8 +82,53 @@ void GDScriptParser::_set_end_statement_error(String p_name) {
 }
 
 bool GDScriptParser::_enter_indent_block(BlockNode *p_block) {
+	if (!_parse_colon()) {
+		return false;
+	}
+
+	if (tokenizer->get_token() != GDScriptTokenizer::TK_NEWLINE) {
+		// be more python-like
+		IndentLevel current_level = indent_level.back()->get();
+		indent_level.push_back(current_level);
+		return true;
+		//_set_error("newline expected after ':'.");
+		//return false;
```

Comment:
Being nitpicky again, but you shouldn't add commented out code (again, I know you are copying from the other function but these things can be fixed in the copy).